src/java/org/apache/cassandra/metrics/ReadRepairMetrics.java

```diff
@@ -29,7 +29,6 @@
     private static final MetricNameFactory factory = new DefaultNameFactory("ReadRepair");
 
     public static final Meter repairedBlocking = Metrics.meter(factory.createMetricName("RepairedBlocking"));
-    public static final Meter repairedAsync = Metrics.meter(factory.createMetricName("RepairedAsync"));
```

Comment:
@adelapena There might be an entry we need to remove for `RepairedAsync` in `cassandra-4.0-jmx.yaml`.

CC @dcapwell

Reply:
Good catch, without that `JMXCompatabilityTest` fails.